support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.jsx

```diff
@@ -83,6 +82,31 @@ const StartDateCopy = ({ startDate, orderIsGift }: {startDate: Option<string>, o
   return null;
 };
 
+const HeroImage = ({ orderIsGift }: {orderIsGift: boolean}) => (
+  <GridPicture
+    sources={[
+        {
+          gridId: !orderIsGift ? 'weeklyLandingHero' : 'gwGiftingPackshot',
```

Comment:
Very small thing but maybe easier to read if not negated?

Reply:
👍 You're right, I'll change these.